teamcity/project_test.go

```diff
@@ -199,6 +199,31 @@ func TestProject_GetRootByName(t *testing.T) {
 	assert.Equal(t, "<Root project>", actual.Name)
 }
 
+func TestProject_ChildProjects(t *testing.T) {
+	client := setup()
+	assert := assert.New(t)
+	_ = createTestProject(t, client, testProjectId)
+	sut := client.Projects
+	actual, err := sut.GetByName(testProjectId)
+	require.NoError(t, err)
+
+	assert.Equal(0, actual.ChildProjects.Count)
+	child1 := createTestProjectWithParent(t, client, "ChildProjectTest1", testProjectId)
+	child2 := createTestProjectWithParent(t, client, "ChildProjectTest2", testProjectId)
+
+	actual, err = sut.GetByName(testProjectId)
+	require.NoError(t, err)
+	cleanUpProject(t, client, testProjectId)
+
+	assert.Equal(2, actual.ChildProjects.Count)
+	childProjects := make(map[string]string, 2)
+	for _, i := range actual.ChildProjects.Items {
+		childProjects[i.ID] = i.Name
+	}
```

Comment:
What would be this `Name` collection for? Did you intend on asserting those names?
Either way the ID assertion is enough, up to you if you want to:
1- Assert the names
or
2- Remove the block

Reply:
Thanks, I forgot to add assertions